### PR TITLE
feat: add new tables and columns, update schema and models

### PR DESCRIPTION
**Pull request by jeanevabejar**
Tables:
- user
- user_profile
- workforce
- workforce_profile
- item
- payment
- order
- order_specification
- artwork
- job

### CODE REVIEW

File: app/models/user.rb
Comment (hugostanley) [inline app/models/user.rb:1]: nit: indentation

File: app/models/workforce.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+        enum role: { admin: 0, illustrator: 1 }
+
+        validates :role, presence: true, inclusion: { in: roles.keys }
+
+        has_one :workforce_profile
+        has_many :job
+        has many :artwork, through :job
```
Comment (hugostanley) [inline app/models/workforce.rb]: nit: indent

**Comment by hugostanley** [comment]
Reminder: commit early commit often.

Next time, much better to have tons of open PRs than one big PR that targets different contexts. 


Other than that. 

LGTM

File: app/models/user_profile.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class UserProfile < ApplicationRecord
```
Comment (louisedungca) [inline app/models/user_profile.rb:1]: model already created. pls check feat/user branch

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
```
Comment (louisedungca) [inline app/models/user.rb:1]: model already created. pls check feat/user branch

File: app/models/workforce.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+class Workforce < ApplicationRecord
```
Comment (louisedungca) [inline app/models/workforce.rb:1]: model already created. pls check feat/workforce branch

File: app/models/workforce_profile.rb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+class WorkforceProfile < ApplicationRecord
```
Comment (louisedungca) [inline app/models/workforce_profile.rb:1]: model already created. pls check feat/workforce branch

File: db/migrate/20240515093336_create_payments.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+class CreatePayments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :payments do |t|
```
Comment (louisedungca) [inline db/migrate/20240515093336_create_payments.rb:3]: config enum in migration too. see this one for ref (lines 3 & 34): https://github.com/hugostanley/toonverse_be/blob/feat/users/db/migrate/20240515051316_devise_token_auth_create_workforces.rb

File: app/models/payment.rb [hunk 7]
```diff
@@ -0,0 +1,9 @@
+class Payment < ApplicationRecord
+  belongs_to :user
+  belongs_to :item
+  has_many :order
+
+  enum status: { pending: 0, paid: 1, cancelled: 2, refunded: 3}
```
Comment (louisedungca) [inline app/models/payment.rb]: change enum values to string. see line 40 for ref: https://github.com/hugostanley/toonverse_be/blob/feat/users/app/models/workforce.rb

**Pull request closed without merging**